app.go

```diff
@@ -81,10 +81,7 @@ func env(key string) string {
 }
 
 func waitUntilNexusBecomesReady(url string, username string, password string) error {
```

Comment:
Unused parameters:
All of the given parameters are no longer used. Shouldn't these be removed as well?

Reply:
Yes, they should. I'll fix that.